### ISSUE TITLE
A11y_AzureToolsStorageExplorer_Application_ScreenReader:  VoiceOver not announces control type of the controls(Application, Data Explorers, Transfers controls).

### ISSUE THREAD

**Issue by Suraj1200499** [microsoft/AzureStorageExplorer#1970]
“[Check out Accessibility Insights!](https://accessibilityinsights.io/) Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11y_AzureToolsStorageExplorer; #A11yMAS; #A11YTCS; #DesktopApp; #MAS1.3.1; #VoiceOver; #Mac;  

**Environment Details:**
Application Name: Azure Tools Storage Explorer
Application Version: 1.10.0
MAC Version 10.14.6 Beta(18G29g)

**Repro Steps:**
1. Launch Storage Explorer
2. Navigate to Setting menu on left pane and click on it.
3. Navigate to Application on right pane and  click on it.
4. "Application" screen gets opened.
5. Start VoiceOver and navigate to "Application, Data Explorers, Transfers controls" using Tab key and Verify VoiceOver not announces announces control type for those controls.

**​Actual:​**
While navigating on the screen using Tab key when Voice Over focus is on "Application, Data Explorers, Transfers controls", VoiceOver not announces control type.

**Expected:**
While navigating on the screen using Tab key when Voice Over focus is on "Application, Data Explorers, Transfers controls", as these are Tab/menu items VoiceOver should announce the control type for these controls.

**UserImpact:**
Screen reader users may not be able to know the control type of the controls if screen reader not announces control type for them.

**Recommendation:**
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

**Attachment for Reference:**
[1970_A11y_AzureToolsStorageExplorer_Application_ScreenReader.pptx](https://github.com/microsoft/AzureStorageExplorer/files/3669418/1970_A11y_AzureToolsStorageExplorer_Application_ScreenReader.pptx)

**Comment by JasonYeMSFT** [comment]
merged.

**Comment by MRayermannMSFT** [comment]
@Radha2019 @Charanrajkb @nirmalsahoo @Suraj1200499 @vinodbk197

Please verify whether or not this bug was fixed to your satisfaction in the latest release of Storage Explorer. You can obtain that release from https://www.storageexplorer.com

If the bug has not been fixed to your satisfaction, please:
- Explain how did the fix not meet your expectations.
- Tell us exactly how we can meet your expectations. If there is more than one way to do so, feel free to list the different ways.

**Comment by Suraj1200499** [comment]
@MRayermannMSFT Verified this issue in Microsoft Azure Tools Storage Explorer Application Version: 1.12.0, this issue is no longer reproduced. Hence Closing the bug.

**Comment by fsteffi** [comment]
GithubTags:#Closed;

**Comment by mstechie** [comment]
GitHubTags:#BM-TCS-StorageExplorer-Mac-Sep2019;
